packages/tools/src/utilities/dynamicVolume/getDataInTime.ts

```diff
@@ -0,0 +1,104 @@
+import { utilities, cache } from '@cornerstonejs/core';
+
+/**
+ *
```

Comment:
need couple of sentence of doc before params

Reply:
added summary of feature before params